src/js/ads.js

```diff
@@ -32,12 +32,12 @@ class VideoAds {
 				document.getElementsByTagName("head")[0].appendChild(googleSdkScript);
 			}
 
-			if(googleSdkScript.hasAttribute('data-o-video-script-loaded')) {
+			if (this.sdkScriptLoaded) {
 				resolve();
 			} else {
 
 				googleSdkScript.addEventListener('load', () => {
-					googleSdkScript.setAttribute('data-o-video-script-loaded', true);
+					this.sdkScriptLoaded = true;
```

Comment:
I tend to go with static properties, but either work and I don't have a
strong opinion

On Wed, 27 Jul 2016 at 15:26 Arjun Gadhia <notifications@github.com> wrote:

> In src/js/ads.js
> <https://github.com/Financial-Times/o-video/pull/15#discussion_r72448861>:
>
> >  				resolve();
> >  			} else {
> >
> >  				googleSdkScript.addEventListener('load', () => {
> > -					googleSdkScript.setAttribute('data-o-video-script-loaded', true);
> > +					this.sdkScriptLoaded = true;
>
> Or even just a variable outside the class?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Financial-Times/o-video/pull/15/files/319dc4cf86eb8afdfd530b6dbe3a562a79482c7a..8abcc2e50684c09c1fe6f0095b0ea01838c716a2#r72448861>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA3u3ZJ2lfuzLSW3jAHcLhgT9ei56heQks5qZ2p3gaJpZM4JWJ4I>
> .
>

-- 

------------------------------

*This email was sent by a company owned by Financial Times Group Limited 
("FT Group <http://aboutus.ft.com/corporate-information/#axzz3rajCSIAt>"), 
registered office at Number One Southwark Bridge, London SE1 9HL.  
Registered in England and Wales with company number 879531. This e-mail may 
contain confidential information. If you are not the intended recipient, 
please notify the sender immediately, delete all copies and do not 
distribute it further.  It could also contain personal views which are not 
necessarily those of the FT Group.  We may monitor outgoing or 
incoming emails as permitted by law.*